feature/bgp/addpath/otg_tests/route_propagation_test/route_propagation_test.go

```diff
@@ -460,6 +466,11 @@ func TestBGP(t *testing.T) {
 			}
 
 			dut := ondatra.DUT(t, "dut")
+			t.Log("Configure Network Instance")
+			dutConfNIPath := dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance)
+			dutConfNIPath.Type().Replace(t, telemetry.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE)
+			dutConfNIPath.RouterId().Replace(t, dutPort2.IPv4)
```

Comment:
While router ID is a constant for these tests, keep it as part of the each test case like in the corresponding ATE test (https://github.com/openconfig/featureprofiles/blob/1a438857b49d6b423ff3a694912cd41a0b26afdf/feature/bgp/addpath/ate_tests/route_propagation_test/route_propagation_test.go#L194). The relevant DUT configuration for a particular test case is made more obvious that way.

Reply:
Made the changes